src/status_im/ui/screens/db.cljs

```diff
@@ -167,6 +167,7 @@
                  :group-chat-profile/editing?
                  :networks/selected-network
                  :networks/networks
+                 :networks/manage
```

Comment:
could you elaborate on choosing global state instead of component-level state to manage form state?

Reply:
I was under the impression it was being scoped down to the component but had to add this in to all the key to exist in the db. I was mainly following what the other screens were doing (i.e. `:accounts/create`) to manage inter-screen interaction